### PR TITLE
Updated the generate responses on the question show/ index view. 

### PR DESCRIPTION
**Pull request by Hellrungj**
the response text links are now changed to basic text and updated to generate one after other after on a different line.

### CODE REVIEW

File: app/views/questions/show.html.haml [hunk 1]
```diff
@@ -29,11 +29,12 @@
 	- @responses.each do |response|
 		%tr
 			%td
-				= link_to response.text, response
+				= response.text
 			%td
 				= response.user_id
```
Comment (hstoebel) [inline app/views/questions/show.html.haml]: instead of displaying the user's id, could we show the user's name?

**Comment by hstoebel** [comment]
This looks great! I just have one suggestion: instead of displaying the user's id, could we show the user's name?

**Comment by Hellrungj** [comment]
Fixed the conflicts.

File: app/views/layouts/_breadcrumb.html.haml [hunk 2]
```diff
@@ -58,5 +58,29 @@
     - else
       %li.active= @organization.abbreviation
 
+  - elsif @edit
```
Comment (hstoebel) [inline app/views/layouts/_breadcrumb.html.haml]: I wonder if there is a more general way to tell the user what part of the Q&A controller they are in. maybe look in params to see what controller, action and id is being requested.

File: app/views/questions/index.html.haml [hunk 3]
```diff
@@ -1,26 +1,30 @@
+= render partial: 'layouts/breadcrumb'
 = render :partial => 'questions/search'
 
 %h1 List of Questions
 
 =link_to "Add New Question", new_question_path, class: "btn btn-default"
 
 -if @questions.size == 0
-	%em = No Questions Found
+  %em = No Questions Found
```
Comment (hstoebel) [inline app/views/questions/index.html.haml:7]: If this isn't indented won't it run regardless of @questions.size

File: app/views/questions/show.html.haml [hunk 4]
```diff
@@ -1,90 +1,91 @@
...
-                      .modal-footer
-                        %button.btn.btn-default{"data-dismiss" => "modal", :type => "button"}
-                          Cancel
-                        %button.btn.btn-primary{:type => "button"}
-                          Submit
-              .col-sm-1
-                %td= link_to "Edit", edit_question_path(@question), class: "btn btn-info"
-                -#%input.btn-info.btn-info{:type => "submit", :value => "Delete"}/
+%h3
+  = "Exercise: #{@question.exercise_id}"
```
Comment (hstoebel) [inline app/views/questions/show.html.haml:8]: could we see the name of the exercise rather than its id `@question.exercise.name`

Comment (Hellrungj) [inline app/views/questions/show.html.haml:8]: I tried to change the line to @question.exercise.name and @question.exercise_id.name both broke. I am not how to show the exercise name.

**Comment by Hellrungj** [comment]
I ready again to pull. I tried working on use the params method  of integrating the _breadcrumb but it didn't work at all. Plus, all information I research does really support the params method.

**Comment by hstoebel** [comment]
cool! Let's review in class tomorrow!

File: app/controllers/questions_controller.rb [hunk 5]
```diff
@@ -31,6 +31,18 @@ def show
     authorize! :read, @question
     @responses = Response.all.where(question_id: params[:id])
     @response = Response.new
+    @title = ''
```
Comment (hstoebel) [inline app/controllers/questions_controller.rb]: consider the `truncate` helper to simplify this. http://api.rubyonrails.org/classes/String.html#method-i-truncate

File: app/views/questions/show.html.haml [hunk 6]
```diff
@@ -1,92 +1,90 @@
...
-              - @responses.each do |response|
-                %tr
-                  %td
-                    = link_to response.text, response
-                  %td
-                    = response.created_at.strftime("%m-%d-%Y")
-                    %br/
-                    %br/
+%h3
+  = "User: #{@question.user_id}"
```
Comment (hstoebel) [inline app/views/questions/show.html.haml]: get the user's name: `"#{@question.user.first_name} #{@question.user.last_name}"`

**Pull request merged**